Python/ast_unparse.c

```diff
@@ -654,11 +662,29 @@ build_fstring_body(asdl_expr_seq *values, bool is_format_spec)
     return PyUnicodeWriter_Finish(body_writer);
 }
 
+static int
+append_templatestr(PyUnicodeWriter *writer, expr_ty e)
+{
+    int result = -1;
+    PyObject *body = build_ftstring_body(e->v.TemplateStr.values, 0);
+    if (!body) {
+        return -1;
+    }
+
+    if (-1 != append_charp(writer, "t") &&
+        -1 != append_repr(writer, body))
```

Comment:
```suggestion
    if (append_charp(writer, "t") != -1 &&
        append_repr(writer, body) != -1)
```
Weird ordering through outt the changes, is there a specific reason??

Reply:
This is https://en.wikipedia.org/wiki/Yoda_conditions and there are preferences over whether to use them or not.  PEP-7 doesn't say that it's forbidden so it's up to the module's conventions. If it's done elsewhere in this module, it's fine to keep it as is.

Reply:
No specific reason other than consistency with the rest of `ast_unparse.c`.